polkadot/runtime/parachains/src/builder.rs

```diff
@@ -839,33 +842,9 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 		// Mark all the used cores as occupied. We expect that there are
 		// `backed_and_concluding_paras` that are pending availability and that there are
 		// `used_cores - backed_and_concluding_paras ` which are about to be disputed.
-		let now = frame_system::Pallet::<T>::block_number() + One::one();
 
 		let mut core_idx = 0u32;
 		let elastic_paras = &builder.elastic_paras;
-		// Assign potentially multiple cores to same parachains,
```

Comment:
because this code was only setting the availability cores and we no longer have this in the storage.
we only need the claim queue and the pending availability candidates storage